lib/msal-node/test/client/ManagedIdentityApplication.spec.ts

```diff
@@ -194,6 +199,199 @@ describe("ManagedIdentityApplication unit tests", () => {
         });
     });
 
+    describe("Acquires a token successfully via an Azure Arc Managed Identity", () => {
+        beforeAll(() => {
+            // make a copy
+            process.env = { ...OLD_ENVS };
+
+            process.env["IDENTITY_ENDPOINT"] = "fake_IDENTITY_ENDPOINT";
+            process.env["IMDS_ENDPOINT"] = "fake_IMDS_ENDPOINT";
+        });
+
+        afterAll(() => {
+            // restore old environment
+            process.env = OLD_ENVS;
+        });
+
+        // Azure Arc Managed Identities can only be system assigned
+        describe("System Assigned", () => {
+            let managedIdentityApplication: ManagedIdentityApplication;
+            beforeEach(() => {
+                managedIdentityApplication = new ManagedIdentityApplication(
+                    systemAssignedConfig
+                );
+            });
+
+            test("acquires a token", async () => {
+                expect(ManagedIdentityTestUtils.isAzureArc()).toBe(true);
+
```

Comment:
Seems like there are a lot of random empty lines. Wouldn't hurt to remove them and shorten the file.